integration_tests/conftest.py

```
@@ -50,3 +58,25 @@ def mantra(request, tmp_path_factory):
 @pytest.fixture(scope="session", params=[True])
 def connect_mantra():
     yield from connect_custom_mantra()
+
+
+@contextmanager
+def setup_all(path, base_port):
+    geth_gen = setup_geth(path, base_port)
+    beacon_gen = setup_beacon(path, base_port)
+    validator_gen = setup_validator(path, base_port)
+    geth_instance = next(geth_gen)
```

Pylint (3.3.7)

[refactor] Do not raise StopIteration in generator, use return statement instead

(R1708)